code/modules/sprite_accessories/tails/_accessory_tail.dm

```diff
@@ -10,7 +10,12 @@
 	// Give us a tail if we need one.
 	var/decl/sprite_accessory/tail_data = GET_DECL(accessories[1])
 	if(tail_data?.draw_accessory && !character.get_organ(BP_TAIL))
-		character.add_organ(new /obj/item/organ/external/tail(null, character.get_mob_snapshot()), null, TRUE, FALSE, FALSE, TRUE)
+		var/obj/item/organ/external/tail/new_tail = new(null, character.get_mob_snapshot())
+		// Unclear why the mob snapshot isn't setting these, but
+		// somehow the tail is ending up as the map default species.
+		new_tail.set_species(character.get_species_name())
```

Comment:
I feel like this should probably be investigated further rather than just slapping a stopgap on there...